torch/testing/_internal/custom_op_db.py

```diff
@@ -331,6 +449,23 @@ def _(boxes, scores, iou_threshold):
     result = boxes.new_empty([i0], dtype=torch.int64)
     return result
 
+def numpy_nms_vmap(info, in_dims, boxes, scores, iou_threshold):
+    boxes_bdim, scores_bdim , _ = in_dims
```

Comment:
Ditto here: nms is data-dependent, so I don't think we can write a vmap rule for it